examples/no-code/src/index.css

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
this should be inlined in index.html. No need for a custom CSS preprocessor here.